### PR TITLE
Future sw update

### PR DESCRIPTION
**Pull request by carriganm95**
Update to the generateMP script to put max full match and max tracklet projection arrays into different order. Previously was done alphabetically and is now changed to start with L1PHIA and end with D5PHID. These arrays are only used by the future SW so the ordering should not affect firmware.

### CODE REVIEW

**Review by bryates — APPROVED** [review]
Looks good to me. @carriganm95 I think we'll want a rebase instead of a merger from the master, but otherwise this PR is good to merge once the CI is done.

**Comment by bryates** [comment]
> Looks good to me. @carriganm95 I think we'll want a rebase instead of a merger from the master, but otherwise this PR is good to merge once the CI is done.

Maybe this isn't a big deal for this small PR, since we always squash before merging back into the master anyway.

**Comment by carriganm95** [comment]
> Looks good to me. @carriganm95 I think we'll want a rebase instead of a merger from the master, but otherwise this PR is good to merge once the CI is done.

I think this should be fixed now

**Comment by bryates** [comment]
> > Looks good to me. @carriganm95 I think we'll want a rebase instead of a merger from the master, but otherwise this PR is good to merge once the CI is done.
> 
> I think this should be fixed now

Perfect, thanks!

**Comment by aryd** [comment]
I think this looks fine so I will go ahead and squash and merge.

**Pull request merged**
